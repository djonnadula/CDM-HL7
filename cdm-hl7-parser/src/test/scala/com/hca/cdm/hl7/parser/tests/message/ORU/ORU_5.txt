MSH|^~\&|COCQA1A|COCQA1A|OV|OV|201611091503||ORU^R01|IP_COQ1A_ORU_NURQA.191416.6593|T|2.5||KYA^04446|ER|NE
PID|1||J000425961^^^COCQA1A^MRN|J502566|PARKER^MAY^R||19620105|F||||||||||J00021035470
PV1|1|I|J.8E^J.CCU^7^COCQA1A|EL|||DRBRI^DOCTOR^BRITTANY|||||||||||IN|||||||||||||||||||||||ADM
ORC|NEW||J00021035470-4008-1-20161109-1458-2
OBR|1|4008|J00021035470-4008-1-20161109-1458-2|4008^CPP TESTING FOR MU|4||201611091458|||||||201611091501||1TSQBE8554||CPPMUDOC
OBX|1|TX|hcaMU204^Smoking status for patients 13 years old or older:||1:Current every day smoker||||||N
OBX|2|TX|hcaMU2COG^Cognitive Status:||Alert:Alert and oriented||||||N
OBX|3|TX|hcaMU2FCT^Functional Status:||Bed:Bed bound||||||N
OBX|4|TX|NUR25002^LOC:||AWAKE/ALER:AWAKE/ALERT||||||N
OBX|5|TX|NUR40002AB^Oriented To:|1|TIMES 3:TIMES 3||||||N
OBX|6|TX|NUR35003Q1^Pulse Ox %:||98||||||N
OBX|7|TX|NUR10006^Pulse:||80||||||N
OBX|8|TX|NUR10014^Blood Pressure:||120/80||||||N
OBX|9|TX|NUR10018^Ht-Ft:||5||||||N
OBX|10|TX|NUR10018A^Ht-In:||5||||||N
OBX|11|TX|NUR10020^Wt-Lbs:||152||||||N
OBX|12|TX|NUR10020A^Oz:||6.00||||||N
OBX|13|TX|NUR65002^Moves all extremities:||Y||||||N
OBX|14|TX|NUR98400J^Call Doctor's Office:||ST:Stabbing||||||N
OBX|15|TX|K31010007A^Height ft:||5||||||N
OBX|16|TX|K31010008A^Height in:||5||||||N
OBX|17|TX|K31010006A^Height cm:||1.000||||||N
OBX|18|TX|K31010001A^Weight lb:||152||||||N
OBX|19|TX|K31010002A^Weight oz:||6.00||||||N
OBX|20|TX|K31010003A^Weight kg:||69.00||||||N
OBX|21|TX|K31010005A^Weight source:||Stated:Stated/Reported||||||N
OBX|22|TX|K33010024A^Blood pressure:||120/80||||||N
OBX|23|TX|K33010028A^Blood pressure source:||Manualcuff:Manual cuff||||||N
OBX|24|TX|K33010003A^Temperature F:||89.6||||||N
OBX|25|TX|K33010004A^Temperature source:||Oral:Oral||||||N
OBX|26|TX|K33010016A^Respiratory rate:||18||||||N
OBX|27|TX|K33010018A^Respiratory source:||Observed:Observed||||||N
OBX|28|TX|K33010006A^Pulse:||80||||||N
OBX|29|TX|K33010010A^Pulse source:||Palpation:Palpation||||||N
OBX|30|TX|L35010001A^SPO2 %:||98||||||N
OBX|31|TX|Q47010214A^Numeric pain scale:||5||||||N