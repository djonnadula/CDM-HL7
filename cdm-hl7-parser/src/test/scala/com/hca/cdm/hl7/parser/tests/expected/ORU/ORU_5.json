{
  "0000.COMN": {
    "sending_facility": "COCQA1A",
    "message_control_id": "IP_COQ1A_ORU_NURQA.191416.6593",
    "medical_record_num": "J000425961",
    "medical_record_urn": "J502566",
    "patient_account_num": "J00021035470"
  },
  "0001.MSH": {
    "field_separator": "|",
    "encoding_characters": "^~\\&",
    "sending_application": "COCQA1A",
    "sending_facility": "COCQA1A",
    "receiving_application": "OV",
    "receiving_facility": "OV",
    "date_time_of_message": "201611091503",
    "message_type": {
      "message_code": "ORU",
      "trigger_event": "R01"
    },
    "message_control_id": "IP_COQ1A_ORU_NURQA.191416.6593",
    "processing_id": "T",
    "version_id": "2.5",
    "continuation_pointer": {
      "network_id": "KYA",
      "unit_number": "04446"
    },
    "accept_acknowledgement_type": "ER",
    "application_acknowledgement_type": "NE"
  },
  "0002.PID": {
    "set_id": "1",
    "patient_id_list": {
      "id_num": "J000425961",
      "assigning_authority": "COCQA1A",
      "id_type_code": "MRN"
    },
    "alternate_patient_id": "J502566",
    "patient_name": {
      "family_name": "PARKER",
      "given_name": "MAY",
      "second_and_further_given_names_or_initials_thereof": "R"
    },
    "date_time_of_birth": "19620105",
    "administrative_sex": "F",
    "patient_account_num": "J00021035470"
  },
  "0003.PV1": {
    "set_id": "1",
    "patient_class": "I",
    "assigned_patient_location": {
      "point_of_care": "J.8E",
      "room": "J.CCU",
      "bed": "7",
      "facility": "COCQA1A"
    },
    "admission_type": "EL",
    "attending_doctor": {
      "person_id": "DRBRI",
      "family_name": "DOCTOR",
      "given_name": "BRITTANY"
    },
    "patient_type": "IN",
    "account_status": "ADM"
  },
  "0004.ORC": {
    "order_control": "NEW",
    "filler_order_num": "J00021035470-4008-1-20161109-1458-2"
  },
  "0005.OBR": {
    "set_id": "1",
    "placer_order_num": "4008",
    "filler_order_num": "J00021035470-4008-1-20161109-1458-2",
    "universal_service_id": {
      "id": "4008",
      "text": "CPP TESTING FOR MU"
    },
    "priority": "4",
    "obsv_date_time_num": "201611091458",
    "specimen_received_date_time": "201611091501",
    "ordering_provider": "1TSQBE8554",
    "placer_field_1": "CPPMUDOC"
  },
  "0006.OBX": {
    "set_id": "1",
    "value_type": "TX",
    "obsv_id": {
      "id": "hcaMU204",
      "text": "Smoking status for patients 13 years old or older:"
    },
    "obsv_value": "1:Current every day smoker",
    "obsv_result_status": "N"
  },
  "0007.OBX": {
    "set_id": "2",
    "value_type": "TX",
    "obsv_id": {
      "id": "hcaMU2COG",
      "text": "Cognitive Status:"
    },
    "obsv_value": "Alert:Alert and oriented",
    "obsv_result_status": "N"
  },
  "0008.OBX": {
    "set_id": "3",
    "value_type": "TX",
    "obsv_id": {
      "id": "hcaMU2FCT",
      "text": "Functional Status:"
    },
    "obsv_value": "Bed:Bed bound",
    "obsv_result_status": "N"
  },
  "0009.OBX": {
    "set_id": "4",
    "value_type": "TX",
    "obsv_id": {
      "id": "NUR25002",
      "text": "LOC:"
    },
    "obsv_value": "AWAKE/ALER:AWAKE/ALERT",
    "obsv_result_status": "N"
  },
  "0010.OBX": {
    "set_id": "5",
    "value_type": "TX",
    "obsv_id": {
      "id": "NUR40002AB",
      "text": "Oriented To:"
    },
    "obsv_sub_id": "1",
    "obsv_value": "TIMES 3:TIMES 3",
    "obsv_result_status": "N"
  },
  "0011.OBX": {
    "set_id": "6",
    "value_type": "TX",
    "obsv_id": {
      "id": "NUR35003Q1",
      "text": "Pulse Ox %:"
    },
    "obsv_value": "98",
    "obsv_result_status": "N"
  },
  "0012.OBX": {
    "set_id": "7",
    "value_type": "TX",
    "obsv_id": {
      "id": "NUR10006",
      "text": "Pulse:"
    },
    "obsv_value": "80",
    "obsv_result_status": "N"
  },
  "0013.OBX": {
    "set_id": "8",
    "value_type": "TX",
    "obsv_id": {
      "id": "NUR10014",
      "text": "Blood Pressure:"
    },
    "obsv_value": "120/80",
    "obsv_result_status": "N"
  },
  "0014.OBX": {
    "set_id": "9",
    "value_type": "TX",
    "obsv_id": {
      "id": "NUR10018",
      "text": "Ht-Ft:"
    },
    "obsv_value": "5",
    "obsv_result_status": "N"
  },
  "0015.OBX": {
    "set_id": "10",
    "value_type": "TX",
    "obsv_id": {
      "id": "NUR10018A",
      "text": "Ht-In:"
    },
    "obsv_value": "5",
    "obsv_result_status": "N"
  },
  "0016.OBX": {
    "set_id": "11",
    "value_type": "TX",
    "obsv_id": {
      "id": "NUR10020",
      "text": "Wt-Lbs:"
    },
    "obsv_value": "152",
    "obsv_result_status": "N"
  },
  "0017.OBX": {
    "set_id": "12",
    "value_type": "TX",
    "obsv_id": {
      "id": "NUR10020A",
      "text": "Oz:"
    },
    "obsv_value": "6.00",
    "obsv_result_status": "N"
  },
  "0018.OBX": {
    "set_id": "13",
    "value_type": "TX",
    "obsv_id": {
      "id": "NUR65002",
      "text": "Moves all extremities:"
    },
    "obsv_value": "Y",
    "obsv_result_status": "N"
  },
  "0019.OBX": {
    "set_id": "14",
    "value_type": "TX",
    "obsv_id": {
      "id": "NUR98400J",
      "text": "Call Doctor's Office:"
    },
    "obsv_value": "ST:Stabbing",
    "obsv_result_status": "N"
  },
  "0020.OBX": {
    "set_id": "15",
    "value_type": "TX",
    "obsv_id": {
      "id": "K31010007A",
      "text": "Height ft:"
    },
    "obsv_value": "5",
    "obsv_result_status": "N"
  },
  "0021.OBX": {
    "set_id": "16",
    "value_type": "TX",
    "obsv_id": {
      "id": "K31010008A",
      "text": "Height in:"
    },
    "obsv_value": "5",
    "obsv_result_status": "N"
  },
  "0022.OBX": {
    "set_id": "17",
    "value_type": "TX",
    "obsv_id": {
      "id": "K31010006A",
      "text": "Height cm:"
    },
    "obsv_value": "1.000",
    "obsv_result_status": "N"
  },
  "0023.OBX": {
    "set_id": "18",
    "value_type": "TX",
    "obsv_id": {
      "id": "K31010001A",
      "text": "Weight lb:"
    },
    "obsv_value": "152",
    "obsv_result_status": "N"
  },
  "0024.OBX": {
    "set_id": "19",
    "value_type": "TX",
    "obsv_id": {
      "id": "K31010002A",
      "text": "Weight oz:"
    },
    "obsv_value": "6.00",
    "obsv_result_status": "N"
  },
  "0025.OBX": {
    "set_id": "20",
    "value_type": "TX",
    "obsv_id": {
      "id": "K31010003A",
      "text": "Weight kg:"
    },
    "obsv_value": "69.00",
    "obsv_result_status": "N"
  },
  "0026.OBX": {
    "set_id": "21",
    "value_type": "TX",
    "obsv_id": {
      "id": "K31010005A",
      "text": "Weight source:"
    },
    "obsv_value": "Stated:Stated/Reported",
    "obsv_result_status": "N"
  },
  "0027.OBX": {
    "set_id": "22",
    "value_type": "TX",
    "obsv_id": {
      "id": "K33010024A",
      "text": "Blood pressure:"
    },
    "obsv_value": "120/80",
    "obsv_result_status": "N"
  },
  "0028.OBX": {
    "set_id": "23",
    "value_type": "TX",
    "obsv_id": {
      "id": "K33010028A",
      "text": "Blood pressure source:"
    },
    "obsv_value": "Manualcuff:Manual cuff",
    "obsv_result_status": "N"
  },
  "0029.OBX": {
    "set_id": "24",
    "value_type": "TX",
    "obsv_id": {
      "id": "K33010003A",
      "text": "Temperature F:"
    },
    "obsv_value": "89.6",
    "obsv_result_status": "N"
  },
  "0030.OBX": {
    "set_id": "25",
    "value_type": "TX",
    "obsv_id": {
      "id": "K33010004A",
      "text": "Temperature source:"
    },
    "obsv_value": "Oral:Oral",
    "obsv_result_status": "N"
  },
  "0031.OBX": {
    "set_id": "26",
    "value_type": "TX",
    "obsv_id": {
      "id": "K33010016A",
      "text": "Respiratory rate:"
    },
    "obsv_value": "18",
    "obsv_result_status": "N"
  },
  "0032.OBX": {
    "set_id": "27",
    "value_type": "TX",
    "obsv_id": {
      "id": "K33010018A",
      "text": "Respiratory source:"
    },
    "obsv_value": "Observed:Observed",
    "obsv_result_status": "N"
  },
  "0033.OBX": {
    "set_id": "28",
    "value_type": "TX",
    "obsv_id": {
      "id": "K33010006A",
      "text": "Pulse:"
    },
    "obsv_value": "80",
    "obsv_result_status": "N"
  },
  "0034.OBX": {
    "set_id": "29",
    "value_type": "TX",
    "obsv_id": {
      "id": "K33010010A",
      "text": "Pulse source:"
    },
    "obsv_value": "Palpation:Palpation",
    "obsv_result_status": "N"
  },
  "0035.OBX": {
    "set_id": "30",
    "value_type": "TX",
    "obsv_id": {
      "id": "L35010001A",
      "text": "SPO2 %:"
    },
    "obsv_value": "98",
    "obsv_result_status": "N"
  },
  "0036.OBX": {
    "set_id": "31",
    "value_type": "TX",
    "obsv_id": {
      "id": "Q47010214A",
      "text": "Numeric pain scale:"
    },
    "obsv_value": "5",
    "obsv_result_status": "N"
  }
}